### ISSUE TITLE
error in eval loss & ppl

### ISSUE THREAD

**Issue by dar-tau** [lucidrains/reformer-pytorch#147]
Hey!
https://github.com/lucidrains/reformer-pytorch/blob/2cbc36bb280c0a6de46d838baad11a533a015fc3/pretraining/self-supervised.py#L327

you are dividing the eval_loss & perplexity each time you pass through the loop

Cheers,
Guy :)
